src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -415,6 +445,57 @@ private function getAbsoluteFilePath(string $filePath): string
         return sprintf('%s/%s', $this->getStubsDirectory(), $filePath);
     }
 
+    /**
+     * @param list<Node\Name> $nameNodes
```

Comment:
Let's please add a paragraph explaining why we're doing this, and perhaps an issue ID, if it's something we can get rid of, with new PHPStorm stubs

Reply:
There's no PHPStubs issue :(

Reply:
Added comment.